shared/dev/dumb-sheet/render.desktop.js

```diff
@@ -8,19 +8,41 @@ import type {Props} from './render'
 import {Box, Text, Input, BackButton} from '../../common-adapters'
 import {globalStyles} from '../../styles'
 
-class Render extends Component<void, Props, any> {
+class DumbSheetRender extends Component<void, Props, any> {
   _onFilterChange: (a: any) => void;
+  _onNext: (a: any, offset: 1 | -1) => void;
+  _box: any;
 
   constructor (props: Props) {
     super(props)
 
     this._onFilterChange = debounce(filter => {
-      this.props.onDebugConfigChange({
-        dumbFilter: filter,
-      })
+      this.props.onDebugConfigChange({dumbFilter: filter})
     }, 300)
   }
 
+  _onNext = (key, offset) => {
+    const keys = Object.keys(dumbComponentMap).map(k => k.toLowerCase()).sort()
+    const idx = keys.indexOf(key.toLowerCase())
+
+    if (idx === -1) {
+      return
+    }
+
+    const nextIdx = idx + offset
+
+    if (nextIdx < 0 || nextIdx >= keys.length) {
+      return
+    }
+
+    const filter = `'${keys[nextIdx]}':50`
+
+    this.props.onDebugConfigChange({dumbFilter: filter})
+    setImmediate(() => {
```

Comment:
it is, really just while you're doing the left/right clicking stuff. i don't think we want to necessarily always scroll up which componentDidUpdate will do